notebooks/MAST/plot_analyze_with_lightkurve/plot_analyze_with_lightkurve.ipynb

```diff
@@ -0,0 +1,987 @@
+{
```

Comment:
<p>Apparently there is a built-in way to do this:</p><p><a href="https://docs.lightkurve.org/reference/api/lightkurve.LightCurve.select_flux.html" target="_blank">https://docs.lightkurve.org/reference/api/lightkurve.LightCurve.select_flux.html</a> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>